extensions/amp-story/1.0/amp-story.js

```diff
@@ -2269,7 +2269,8 @@ export class AmpStory extends AMP.BaseElement {
     const historyNavigationPath =
       getHistoryState(this.win, HistoryState.NAVIGATION_PATH);
     if (historyNavigationPath) {
-      this.storyNavigationPath_ = historyNavigationPath;
+      this.storyNavigationPath_ =
+          this.storyNavigationPath_.concat([historyNavigationPath]);
```

Comment:
This is a change in behavior

Reply:
Actually, can we leave this code the way it was initially and just change the type of `getHistoryState` to `{*}` (which is what it actually should be)?

Reply:
Tried this, but I'm seeing new errors elsewhere in the code:
```js
extensions/amp-story/1.0/amp-story.js:885: ERROR - inconsistent return type
found   : *
required: string
      return historyPage;
             ^^^^^^^^^^^

extensions/amp-story/1.0/amp-story.js:2272: ERROR - assignment to property storyNavigationPath_ of AmpStory
found   : *
required: (Array<string>|null)
      this.storyNavigationPath_ = historyNavigationPath;
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
As discussed in person, I've undone the code changes and fixed the type info. Done in https://github.com/ampproject/amphtml/pull/21618/commits/98a88e70b4c2b6c229b1d987d37a8aca65eeda51.